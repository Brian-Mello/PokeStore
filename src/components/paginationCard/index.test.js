import React from 'react';
import renderer from 'react-test-renderer';
import PaginationCard from '.';

describe('testando integridade da Pagination card component', () => {
    it(' Pagination card snapshot', () => {
        const paginationCard = renderer.create(<PaginationCard/>).toJSON();
        expect(paginationCard).toMatchSnapshot();
    });
}); 