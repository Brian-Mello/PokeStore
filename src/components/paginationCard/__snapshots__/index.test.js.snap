// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando integridade da Pagination card component  Pagination card snapshot 1`] = `
<div
  className="sc-AxiKw kdNFlC"
>
  <div
    className="sc-AxgMl celCAs"
  >
    <div
      className="sc-AxheI kZkVEU"
    />
    <div
      className="sc-AxmLO fdRSyf"
    >
      <b />
    </div>
    <div
      className="sc-Axmtr hShZhH"
    />
  </div>
</div>
`;
